pkg/common/jobcontroller/service.go

```diff
@@ -138,11 +139,13 @@ func (jc *JobController) GetServiceSlices(services []*v1.Service, replicas int,
 			logger.Warningf("Error when strconv.Atoi: %v", err)
 			continue
 		}
-		if index < 0 || index >= replicas {
+		if index < 0 {
 			logger.Warningf("The label index is not expected: %d", index)
-		} else {
+		} else if index < replicas {
 			serviceSlices[index] = append(serviceSlices[index], service)
```

Comment:
@gaocegege In which situations, one index can be corresponding to multiple pods?

Reply:
There is no such situation, I think.

Reply:
IMHO, we can make it just return `[]*v1.Pod` instead of `[][]*v1.Pod`. `[][]*v1.Pod` is sort of confusing and makes code harder to understand.